common/src/Preferences.cpp

```diff
@@ -192,6 +192,40 @@ namespace TrenchBroom {
             return pref;
         }
 
+        Preference<bool> ShowEntityClassnames(IO::Path("Map view/Show entity classnames"), true);
+        Preference<bool> ShowGroupBounds(IO::Path("Map view/Show group bounds"), true);
+        Preference<bool> ShowBrushEntityBounds(IO::Path("Map view/Show brush entity bounds"), true);
+        Preference<bool> ShowPointEntityBounds(IO::Path("Map view/Show point entity bounds"), true);
+        Preference<bool> ShowPointEntityModels(IO::Path("Map view/Show point entity models"), true);
+
+        QString faceRenderModeTextured() { return "textured"; }
+        QString faceRenderModeFlat() { return "flat"; }
+        QString faceRenderModeSkip() { return "skip"; }
+        Preference<QString> FaceRenderMode(IO::Path("Map view/Face render mode"), "textured");
```

Comment:
Could you not use an enum instead of strings?

Reply:
As far as how we interact with these from C++, yeah, it should be an enum instead of these `faceRenderMode*()` functions, but this is a bit of work. I can file a separate issue. (it would also apply to the Theme preference - I copied the setup of that for these `faceRenderMode*()` functions).

As far as the JSON representation, strings are the correct way to do it, rather than just writing an int like `"Map view/Face render mode":5`

Reply:
Agree with the json side of things, but it would be cool if we could use an enum on the c++ side. I wasn't aware we already use this pattern though, so for now leave it as is.